lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -131,7 +131,7 @@
 
                     if( !this.model.get('country') ){
                         $countryLabel.addClass('error');
-                        $errorMessageContainer.append("<li>" + msg + "</li>");
+                        HtmlUtils.append($errorMessageContainer, HtmlUtils.joinHtml("<li>", msg, "</li>"));
```

Comment:
This is now escaping ``msg`` when it wasn't before. It is also going to be escaping the HTML strings. This needs to be more like:

    HtmlUtils.joinHtml(
        HtmlUtils.HTML('<li>'), 
        HtmlUtils.HTML(msg),
        HtmlUtils.HTML('</li>')
    )